src/components/Modals/PrintModal.vue

```diff
@@ -15,121 +15,105 @@ const updateLocalShowNotification = (value) => {
   emit('update:show-modal', value)
 }
 
-// INITIALISE TEMPLATE REFS
-const selectAll = ref(null)
-const intermediate = ref(null)
-const refs = ref([])
-onMounted(() => {
-  refs.value = Array(printOptions.length)
-    .fill(null)
-    .map(() => ref(null))
-})
-function setRef(index) {
-  return (el) => {
-    refs.value[index] = el
+watch(
+  () => props.showNotification,
```

Comment:
dont't think this is needed in the first place. on dialog close, you could have just emit the event out to update the close status which is taken in by the props. makes no sense to store a prop into a variable and watch it when the props is already reactive